### PR TITLE
Added a 'completeSlider' function that I found handy.

### PR DESCRIPTION
**Pull request by MichaelStevan**
## Status
**READY FOR USE**

## Description
A public function was implemented so that a given slider can be completed on demand.

## Related PRs/Issues
No issues. 
Is a very simple function that works as expected.

## Feedback Reviews

@cortinico

### CODE REVIEW

File: example/build.gradle [hunk 1]
```diff
@@ -2,7 +2,6 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "25.0.3"
```
Comment (cortinico) [inline example/build.gradle]: Please revert this change. Keep only functional code inside the PR

File: slidetoact/build.gradle [hunk 2]
```diff
@@ -34,7 +33,7 @@ dependencies {
 }
 
 buildscript {
-    ext.kotlin_version = '1.1.2-3'
```
Comment (cortinico) [inline slidetoact/build.gradle]: Please revert this change. Keep only functional code inside the PR

File: README.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Or you can download the .AAR artifact [directly from Bintray](https://bintray.co
 After setting up the Gradle dependency, you can use ``SlideToActView`` widgets inside your **XML Layout files**
 
 ```xml
-<com.ncorti.slidetoact.SlideToActView
+<superlemoncandy.myslidetoact.SlideToActView
```
Comment (cortinico) [inline README.md]: ?

File: README.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ SlideToActView sta = (SlideToActView) findViewById(R.id.example);
 By the default, every ``SlideToActView`` widget fits to your app using the ``colorAccent`` and the ``colorBackground`` parameters from your theme. You can customize your ``SlideToActView`` using the following **custom attributes**.
 
 ```xml
-<com.ncorti.slidetoact.SlideToActView
+<superlemoncandy.myslidetoact.SlideToActView
```
Comment (cortinico) [inline README.md]: ?

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 5]
```diff
@@ -1,638 +0,0 @@
-package com.ncorti.slidetoact
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: Why you deleted the file here?

**Comment by MichaelStevan** [comment]
I was using the library on my own project as I waited for the pull request to be accepted and that's why I had changed stuff around and pushed without realizing. 

Hope it wasn't too much of a bother, it's also kind of my first pull request ^^
So yeah, I'm excited for using the new complete function from the actual dependency !

**Comment by cortinico** [comment]
> Hope it wasn't too much of a bother, it's also kind of my first pull request ^^

That's awesome 😍 I really appreciate the effort! And is also cool to see other people contributing to your library. Welcome to the magic world of OSS ;)

> So yeah, I'm excited for using the new complete function from the actual dependency !

I'm going to release a minor version of your library in the upcoming days. Stay tuned ;)
